### PR TITLE
Last second upload of yaml

### PR DESCRIPTION
**Pull request by Jothle12**
Last second blog yaml update!

### CODE REVIEW

**Comment by decause** [comment]
@Jothle12 This is not your litreview1. Please make sure you gave the correct link.

**Comment by Jothle12** [comment]
Whoops! My bad! Now it is correct!

File: scripts/people/Jothle12.yaml [hunk 1]
```diff
@@ -6,3 +6,4 @@
   irc: Jothle12
   name: Jordan Lehman
   rit_dce: jtl7264
+  litreview1: http://jothle12.blogspot.com/2014/02/capability-and-understanding-reading-on.html://github.com/Jothle12/hflossk.git
```
Comment (decause) [inline scripts/people/Jothle12.yaml]: @Jothle12 close! Your URL has one more URL in it than you intended to include, I reckon.

:wrench: Fix it and I'll merge

**Comment by Jothle12** [comment]
...Well this was a big ball of fail on my part. Anyways, it should be done, along with the quiz. Sorry for the flubs.

**Comment by Jothle12** [comment]
_Cough-cough_

**Comment by decause** [comment]
:+1:

**Pull request merged**